contracts/abstract/NotionalV2BaseLiquidator.sol

```diff
@@ -128,7 +128,7 @@ abstract contract NotionalV2BaseLiquidator {
 
         // Will withdraw entire cash balance. Don't redeem local currency here because it has been flash
         // borrowed and we need to redeem the entire balance to underlying for the flash loan repayment.
-        _redeemAndWithdraw(localCurrency, uint96(netNTokens), false);
+        _redeemAndWithdraw(localCurrency, uint96(int96(netNTokens)), false);
```

Comment:
Not totally sure but I'm guessing you changed this because you wanted to bump `contracts/abstract/NotionalV2BaseLiquidator.sol` to solidity 0.8.0 but the current pragma is >0.7.0, is that intended?